src/foam/nanos/test/Test.js

```diff
@@ -185,7 +185,7 @@ foam.CLASS({
 
         ByteArrayOutputStream baos  = new ByteArrayOutputStream();
         PrintStream           ps    = new PrintStream(baos);
-        Interpreter           shell = createInterpreter(x);
+        Interpreter           shell = (Interpreter) createInterpreter(x, null);
```

Comment:
how does this work with JShell?

Reply:
createInterpreter will check the language to create the appropriate interpreter (jShell or beanShell). after that, we can interpret the code.